docs/how-tos/image-data.md

```diff
@@ -140,17 +147,32 @@ From the **Select model** dropdown, select the name of your ML model service (fo
 
 The `filtered-camera` {{< glossary_tooltip term_id="modular-resource" text="modular component" >}} pulls the stream of images from the camera you configured earlier, and applies the vision service to it.
 
-Configure a `filtered-camera` component on your machine, following the [attribute guide in the README](https://github.com/erh/filtered_camera?tab=readme-ov-file#configure-your-filtered-camera) to specify the names of your camera and vision service, and add classification and object detection filters.
+Configure a `filtered-camera` component on your machine, following the [attribute guide in the README](https://github.com/erh/filtered_camera?tab=readme-ov-file#configure-your-filtered-camera).
+Use the name of the camera you configured in the first part of this guide as the `"camera"` to pull images from, and select the name of the vision service you just configured as your `"vision"` service.
+Then add all or some of the labels your ML model uses as classifications or detections in `"classifications"` or `"objects"`.
+
+For example, if you are using the `face-detection` model, you could use a configuration like the following to only capture images when a face is detected in your camera stream:
```

Comment:
```suggestion
For example, if you are using the `face-detection` model, you could use a configuration like the following to only capture images when the detector detects a face in your camera stream with 80% confidence or higher:
```
[opt] so they know what the .8 means 

BTW I’m still confused by the buffer window seconds attribute—since it’s optional I never used it but do you know? If so maybe explain? Or remove if not needed?